src/SearchResponse.swift

```diff
@@ -12,12 +12,30 @@ import Pelias
 /// Represents a response for a request executed by 'MapzenSearch'
 @objc(MZSearchResponse)
 public class SearchResponse : NSObject {
+
   let peliasResponse: PeliasResponse
 
   private lazy var internalParsedResponse: ParsedSearchResponse? =  { [unowned self] in
     guard let peliasParsedResponse = self.peliasResponse.parsedResponse else { return nil }
     return ParsedSearchResponse.init(peliasParsedResponse)
   }()
+
+  private static let validTypes: Set = ["Point", "MultiPoint", "LineString", "MultiLineString", "Polygon", "MultiPolygon", "GeometryCollection", "Feature", "FeatureCollection"]
+
+  private lazy var internalIsGeoJson: Bool = { [unowned self] in
```

Comment:
Unsure if this is the right approach.

If we don't get valid GeoJSON back - does it make sense to return a formed response object? Maybe instead return an error? Would require changing the closure parameters (honestly I thought there was an error object coming back).